addons/armaos/functions/fnc_os_date.sqf

```diff
@@ -1,38 +1,31 @@
-params ["_options", "_consoleInput"];
+/**
+ * Prints/outputs the ingame date and time on a given computer.
+ * Also returns informations about the success of the command.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Options <[STRING]>
+ *
+ * Results:
+ * 1: Informations/Date and Time <[STRING]>
+ */
 
-_result = [];
+params ["_computer", "_options"];
 
-_optionsCount = count _options;
+if (count _options >= 1) exitWith {["   Command: date has no options"];};
 
-scopeName "main";
+_date = date;
+_year = _date select 0;
+_month = _date select 1;
+_day = _date select 2;
 
-switch (true) do
-{
-	case (_optionsCount >= 1):
-	{
-		//hint "Case 1";
-		
-		_result = ["   Command: date has no options"];
-		_result breakOut "main";
-	};
-	case (_optionsCount == 0):
-	{
-		//hint "Case 2";
+if (_month < 10) then {_month = format["0%1", _month]};
+if (_day < 10) then {_day = format["0%1", _day]};
 
-		_date = date;
-		_year = _date select 0;
-		_month = _date select 1;
-		_day = _date select 2;
+_time = [daytime, "HH:MM:SS"] call BIS_fnc_timeToString; // 07:21:12
```

Comment:
private

Reply:
changed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763